### ISSUE TITLE
Freshly placed nrl-gate doesn't receive initial value

### ISSUE THREAD

**Issue by minedaddy** [nidayand/node-red-contrib-norelite#6]
Insert a nrl-source node and select or create a config and give it a name. Uncheck the "toggle" checkbox and set the default value of the source node to 1. Deploy and observe that the node value is 1. Insert a nrl-gate node and select the same config. Create a rule testing the source node name with "1".  Cut the new gate node then paste back. Deploy and observe that the gate node shows "Initializing' and is in the "closed" state as any inputs are passed to the second output. The gate node should be "open" reflecting the source node's initial value. Changing the state of the source node to 0 will keep the gate node "closed" but now the "Initializing" status is replaced with the correct state and the nodes will all work as expected from this point on. Cutting then re-pasting the gate node and deploying re-introduces the problem.

Basically, a newly inserted gate node only initializes after the state of the source node changes.

**Comment by minedaddy** [comment]
It looks like the method in nrl-source which ensures values are only sent once should resend if the flow id re-deployed.
